backend/s3/s3.go

```diff
@@ -1039,6 +1039,12 @@ func s3Connection(opt *Options) (*s3.S3, *session.Session, error) {
 	def := defaults.Get()
 	def.Config.HTTPClient = lowTimeoutClient
 
+	// start a new AWS session
+	awsSession, err := session.NewSession()
+	if err != nil {
+		return nil, nil, fmt.Errorf("NewSession: %v", err)
```

Comment:
In rclone we are using the pkg/errors package for error wrapping so that should be

```go
return nil, nil, errors.Wrap(err, "NewSession")
```

Reply:
Fixed, thanks.